src/main/java/racingcar/model/RacingCar.java

```diff
@@ -0,0 +1,41 @@
+package racingcar.model;
+
+import java.util.Objects;
+
+public class RacingCar {
+    private RacingCarPosition status;
+    private final MovableStrategy movableStrategy = new RandomMovableStrategy();
+
+    public RacingCar() {
+        this.status = new RacingCarPosition();
+    }
+
+    public RacingCar(RacingCarPosition status) {
+        this.status = status;
+    }
+
+    public void move() {
+        this.status = status.add();
+    }
+
+    public int getStatus() {
+        return this.status.getValue();
+    }
+
+    public boolean isMovable() {
+        return movableStrategy.isMovable();
+    }
```

Comment:
움직일지에 대한 선택도 RacingCar가 할 수 있도록 아래와 같은 형태는 어떨까요?
```suggestion
    public void move(MovableStrategy strategy) {
        if (movableStrategy.isMovable()) {
            this.status = status.add();
        }
    }
```

추가적으로 현재는 MovableStrategy를 내부 상태로 갖고있는데 이게 꼭 필요한 상태인지 고민해보면 좋을 것 같아요 :)
move 호출 시점에 주입받는건 어떻게 생각하시나요?

Reply:
@ydh6226 move 안에 isMovable를 넣으면 랜덤값에 대한 테스트는 어떻게 작성해야할까요?